doc/sphinx/language/core/basic.rst

```diff
@@ -150,10 +151,10 @@ Other tokens
   Here are the character sequences that |Coq| directly defines as tokens
   without using :cmd:`Notation`::
```

Comment:
This should be reformulated to also talk about the fact that tokens defined in plugins are not listed.

Reply:
Maybe simpler for me if you make a suggestion.

Reply:
```rst
Keywords
  The following character sequences are keywords defined in the main Coq grammar
  that cannot be used as identifiers (even when starting Coq with the `-noinit`
  command-line flag)::

     _ Axiom CoFixpoint Definition Fixpoint Hypothesis Parameter Prop
     SProp Set Theorem Type Variable as at cofix else end
     fix for forall fun if in let match return then where with

  The following are keywords defined in notations or plugins loaded in the :term:`prelude`::

    IF by exists exists2 using

  Note that loading additional modules or plugins may expand the set of reserved
  keywords.

Other tokens
  The following character sequences are tokens defined in the main Coq grammar
  (even when starting Coq with the `-noinit` command-line flag)::

    ! #[ % & ' ( () ) * + , - ->
    . .( .. ... / : ::= := :> :>> ; < <+ <- <:
    <<: <= = => > >-> >= ? @ @{ [ ] _
    `( `{ { {| | }

  The following character sequences are tokens defined in notations or plugins
  loaded in the :term:`prelude`::

    ** [= |- || ->

  Note that loading additional modules or plugins may expand the set of defined
  tokens.
```

Plus apply this patch so that "prelude" is defined:

```diff
diff --git a/doc/sphinx/language/coq-library.rst b/doc/sphinx/language/coq-library.rst
index acdd4408ed..d8fa5c4749 100644
--- a/doc/sphinx/language/coq-library.rst
+++ b/doc/sphinx/language/coq-library.rst
@@ -9,7 +9,7 @@ The |Coq| library
 
 The |Coq| library has two parts:
 
-  * **The basic library**: definitions and theorems for
+  * **The :gdef:`prelude`\**: definitions and theorems for
     the most commonly used elementary logical notions and
     data types. |Coq| normally loads these files automatically when it starts.
 ```

Note that my listing of keywords / tokens defined in the prelude is probably incomplete.

Reply:
How about saying  "additional keywords" and "additional character sequences" for the description of the prelude items?